README.md

```diff
@@ -81,6 +79,25 @@ cp .env.example .env
 
 After filling in the values in `.env`, you are ready to run the project!
 
+### Authentication Setup with Clerk
+
+To setup authentication with Clerk, follow the guide [here](https://clerk.com/docs/quickstarts/setup-clerk).
+
+TLDR: You need to make an account and update the following environment variables in `.env`. Note: you do not need to modify NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY.
+
+```
+CLERK_SECRET_KEY=
+```
+
+For the following environment variables, keep them the same for now, unless you're working on authentication. When you update them you should update them in `.env.example` as well
+
+```
+NEXT_PUBLIC_CLERK_SIGN_IN_URL=
+NEXT_PUBLIC_CLERK_SIGN_UP_URL=
+NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL=
+NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL=
+```
+
```

Comment:
Do you know what headers need to be set that Clerk uses? Might want to add a section on that for if people want to test their protected endpoints using trpc panel (although maybe just testing it with publicProcedure is easier)

Reply:
Not too familiar with what headers need to be used, is this some additional setup TRPC panel requires for cross origin requests?

Reply:
this question was dumb -- I was thinking like the panel is postman, but since it's same domain it can also use the auth